views/challenge/create.php

```diff
@@ -55,6 +55,7 @@
             <?= $form->field($model, 'any_reward_asset')->radio(['onclick' => 'document.getElementById("challenge-no_rewarding").checked = false;
                 document.getElementById("challenge-specific_reward_asset").checked = false; 
```

Comment:
alot of js here , all this needs to be moved to `humhub-modules-xcoin/resources/js/xcoin.js`